.github/actions/custom-actions/aselo_development_custom/action.yml

```diff
@@ -95,12 +112,16 @@ runs:
     - name: Add IWF_API_URL
       run: echo "IWF_API_URL=${{ env.IWF_API_URL }}" >> .env
       shell: bash
-    # - name: Add IWF_API_ENVIRONMENT
-    #   run: echo "IWF_API_ENVIRONMENT=${{ env.IWF_API_ENVIRONMENT }}" >> .env
-    #   shell: bash
-    # - name: Add IWF_API_COUNTRY_CODE
-    #   run: echo "IWF_API_COUNTRY_CODE=${{ env.IWF_API_COUNTRY_CODE }}" >> .env
-    #   shell: bash
+
+    - name: Add AS_DEV_IWF_SECRET_KEY
+      run: echo "AS_DEV_IWF_SECRET_KEY=${{ env.AS_DEV_IWF_SECRET_KEY }}" >> .env
```

Comment:
This will deploy an env var called `AS_DEV_IWF_SECRET_KEY` to the actual Serverless instance. Is that intended? How will the code know which variable to use on each account?

Reply:
@GPaoloni, the env is for staging, which I believe is what the Aselo Development was created for. The prod env will be deployed to the prod account of helplines that would use the feature in production. @stephenhand, I am not sure if I answered correctly.